server.py

```diff
@@ -25,6 +25,11 @@
         statement = fl.read()
     questions[i] = Question(output=output, statement=statement)
 
+@app.get('/')
+def redirect():
+	number = '1'
```

Comment:
using the redirect function would be better?

https://bottlepy.org/docs/dev/tutorial.html#http-errors-and-redirects

Reply:
Changed to redirtect :)
Thanks.